### DISCUSSION TITLE
Limit emoji list.

### DISCUSSION THREAD

**Discussion by amiroslaw** [wezterm/wezterm#4019] [Ideas]
For me Character Selection Mode has too many elements, I use only few emojis. I know that you have categories of the characters like 'SmileysAndEmotion', maybe it would be easy to limit that list in configuration to one or only few groups.

**Comment by bew** [comment]
You can change what is the default group using a keybind:
https://wezfurlong.org/wezterm/config/lua/keyassignment/CharSelect.html

Although it is not possible (at the moment) to restrict only to a few groups.

**Comment by wez** [comment]
I'm not sure what you mean by too many elements. The purpose is to allow you to select from any possible emoji, so it has to be able to show you all of them.  There are only so many that can fit on the screen, so it doesn't show them all at once.
Are you saying that there are too many on the screen?

Note that the default view is ordered by frecency, so the emoji that you use most often will bubble up to the top of the list as you use them.

**Reply by amiroslaw** [reply]
Sorry for the confusion. I didn't mean that there are too many on the screen, but too are many available. I'd like to restrict to a few groups because I know that 95% of them, like flags, I won't use. 

I believe that for people like me who don't remember the names of the emojis and don't need all of them, this will increase usability.

I know about frecency, but that list is deleted after reboot. 

One question, is there an option to add a custom keybinding for moving down/up the cursor in this mode?

**Reply by wez** [reply]
> I know about frecency, but that list is deleted after reboot.

See: https://github.com/wez/wezterm/issues/4047 which is now fixed in `main`

> One question, is there an option to add a custom keybinding for moving down/up the cursor in this mode?

Not today
